backend/database/actionsWithTables/storageActions.js

```diff
@@ -1,32 +1,41 @@
 const { Storage } = require('../tables/storage');
+const { Product } = require('../tables/product');
 
 const addNewStorage = async (data) => {
     return await Storage.create({
         name: data.name,
     });
 };
 
-const returnStorages = async () => {
+const getAllStorages = async () => {
     const storages = await Storage.findAll({
         attributes: ['storage_id', 'name', 'is_deleted'],
         where: {
             is_deleted: false,
-        }
+        },
     });
     return await storages;
 };
 
-const isDeleteStorage = async (data) => {
+const deleteStorage = async (data) => {
     await Storage.update(
         {
             is_deleted: true,
         },
         {
-            where: { storage_id: Object.values(data) },
+            where: { storage_id: data.id },
+        }
+    );
+    await Product.update(
+        {
+            is_deleted: true,
+        },
+        {
+            where: { storage_id: data.id },
```

Comment:
It is necessary to delete products if storage is deleted? Because products can be located in other storages too